src/App.css

```
@@ -85,4 +85,15 @@
 .signin h1{
   color: white;
   text-align: center;
+}
+footer {
+  position: fixed;
+  background-color: #2c393e;
+  color: white;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  padding: 19px;
+  text-align: center;
+  font-size: 18px;
```

```suggestion
    position: fixed;
    background-color: #00000000;
    color: white;
    bottom: 0;
    left: 0;
    right: 0;
    padding: 5px;
    text-align: center;
    font-size: 10px;
    border-top: 1px solid gray;
```

Try this also,

we can update the `.maincontainer` height to `84.5px`.

@yashikajotwani12 WDYT?

---

I have updated as per your review.